src-built-in/components/advancedAppLauncher/src/stores/StoreActions.js

```diff
@@ -238,32 +278,40 @@ function getToolbarStore(done) {
 	});
 }
 
-function _setFolders(cb = Function.prototype) {
-	getStore().setValue({
-		field: "appFolders.folders",
-		value: data.folders
-	}, (error, data) => {
-		if (error) {
-			console.log("Failed to save modified folder list.");
-		} else {
-			cb();
-		}
-	});
-}
-
-function _setValue(field, value, cb) {
+function _setValue(field, value, cb = Function.prototype) {
 	getStore().setValue({
 		field: field,
 		value: value
 	}, (error, data) => {
 		if (error) {
 			console.log("Failed to save. ", field);
+			FSBL.Clients.Logger.error("Advanced App Launcher: Failed to save ", field);
```

Comment:
Can you log the value too? It looks like it might be spamy, but this is an error case.